<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>21</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Start </string>
    </property>
   </widget>
   <widget class="QPushButton" name="redButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>180</red>
          <green>180</green>
          <blue>180</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 180, 180)</string>
    </property>
    <property name="text">
     <string>Red</string>
    </property>
   </widget>
   <widget class="QPushButton" name="blueButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>251</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <italic>true</italic>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 180, 180)</string>
    </property>
    <property name="text">
     <string>Blue</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="turnLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>100</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Simon's Turn </string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>20</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Correct: Press to Continue</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA5_Simon_Game">
    <property name="title">
     <string>A5 - Simon Game</string>
    </property>
   </widget>
   <addaction name="menuA5_Simon_Game"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
